packages/form-js-viewer/test/spec/Form.spec.js

```diff
@@ -752,11 +829,19 @@ describe('Form', function() {
       expect(event.data).to.exist;
       expect(event.errors).to.exist;
 
+      expect(event.data).to.eql({
+        creditor: '',
+        invoiceNumber: '',
+        amount: 456,
+        approved: false,
+        approvedBy: '',
```

Comment:
Interesting case: Is a `string` value if empty `''` or `null`?

Reply:
According to https://github.com/bpmn-io/form-js/pull/177 only `number` and `select` (and `radio` actually but you cannot unset it) have no empty value.

Reply:
I anticipate we'll get more feedback on this in the future. But for the moment, should be fine (and less breaking in terms of change).

Reply:
https://github.com/bpmn-io/form-js/pull/177 only captures what we thought what was good in the past. You could validate my comment with Martin an platform stakeholders, too.

Reply:
I've discussed this approach with @marstamm. I also added him as a reviewer.